apps/api/src/app/integrations/usecases/create-integration/create-integration.command.ts

```diff
@@ -19,5 +19,5 @@ export class CreateIntegrationCommand extends EnvironmentCommand {
   check: boolean;
 
   @IsOptional()
-  userId?: string;
+  userId: string;
```

Comment:
Mmmm... unless this is a mistake or if not you should remove the decorator also.

Reply:
the userId is always passed to this command

Reply:
Then let's remove the `isOptional`. 💪🏻